libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,164 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+import { DateRangePickerInputField } from "./DateRangePickerInputField";
+import { injectIntl, WrappedComponentProps } from "react-intl";
+import { IntlWrapper } from "@gooddata/sdk-ui";
+import DayPickerInput from "react-day-picker/DayPickerInput";
+import moment from "moment";
+import { DateRangePickerInputFieldBody } from "./DateRangePickerInputFieldBody";
+import { convertPlatformDateStringToDate } from "../utils/DateConversions";
+import { TIME_FORMAT } from "../constants/Platform";
+
+interface IDateTimePickerOwnProps {
+    placeholderDate: string;
+    dateFormat: string;
+    onChange: (value: Date) => any;
+    value: Date;
+    dayPickerPropsWithDefaults: any;
+    handleDayClick: any;
+    locale: string;
+    isMobile: boolean;
+    isTimeForStaticPeriodEnabled: boolean;
+    className: string;
+    defaultTime?: string;
+    error?: boolean;
+}
+
+export type DateTimePickerComponentProps = IDateTimePickerOwnProps & WrappedComponentProps;
+
+// eslint-disable-next-line react/display-name
+export const DateTimePickerComponent = React.forwardRef<DayPickerInput, DateTimePickerComponentProps>(
+    (props: DateTimePickerComponentProps, ref) => {
+        const {
+            placeholderDate,
+            value,
+            onChange,
+            dateFormat,
+            dayPickerPropsWithDefaults,
+            handleDayClick,
+            isMobile,
+            isTimeForStaticPeriodEnabled,
+            className,
+            defaultTime = "00:00",
+            error = false,
+        } = props;
+
+        const getFormattedDate = () => {
+            return moment(value).format("YYYY-MM-DD");
+        };
+
+        const getFormattedTime = () => {
```

Comment:
This one as well: I would move this function outside of the component and provide `value` as a parameter.

Reply:
I would follow the same logic as with Date/Calendar. Basically nobody else should use the method as it is connected to internal logic of time related part.